src/services/api-service.ts

```diff
@@ -243,6 +245,50 @@ class ApiService {
     })
     return result
   }
+
+  async getBlockHeights(process: BlockHeightProcessType, chainId?: number) {
+    const query = gql`
+      query GetAllBlockHeights($getBlockHeight: GetAllBlockHeightInput) {
+        getAllBlockHeights(getBlockHeight: $getBlockHeight) {
+          chainId
+          process
+          blockHeight
+          isActive
+        }
+      }
+    `
+    const input = {getBlockHeight: {chainId, process}}
+    const data: BlockHeightResponse = await this.client.request(query, input)
+    return data.getAllBlockHeights
+  }
+
+  async updateBlockHeight<T = any>(
```

Comment:
just wondering what the benefit is of typing this to `any`?

Reply:
actually, I don't think it makes any difference, I just followed the typing used internally in the `rawRequest` function.